### PR TITLE
CLN:Clean up setup.cfg file and flake8 errors on test/test_repo_template/setup.py file

### PR DESCRIPTION
**Pull request by dorothykiz1**

### CODE REVIEW

File: test/test_repo_template/asv_test_repo/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-dummy_value = {dummy_value}
+dummy_value = {dummy_value}  # noqa see #1147
```
Comment (datapythonista) [inline test/test_repo_template/asv_test_repo/__init__.py:3]:
```suggestion
dummy_value = {dummy_value}  # noqa <FLAKE8_ERROR_CODE> This is a template that will be rendered to valid Python
```

File: test/test_repo_template/setup.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 with open('asv_test_repo/build_time_env.py', 'w') as f:
-    f.write("env = {{}}\n".format(repr(dict(os.environ))))
+    f.write("env = {{}}\n".format(repr(dict(os.environ))))  # noqa see #1144
```
Comment (datapythonista) [inline test/test_repo_template/setup.py:6]: Can you simply change the string for `f'env = {os.environ}'`. Or if it doesn't work just `env = {}`. This code doesn't make any sense, it simply writes `env = {}`, but I guess the intention was to have the envionment, so let's try first if that works.

Comment (dorothykiz1) [reply test/test_repo_template/setup.py:6]: Thank you,Yes,ran it locally it just returns `env = {}`

Comment (datapythonista) [reply test/test_repo_template/setup.py:6]: I see, can you try with `f'env = {os.environ}'` which makes more sense.

File: test/test_repo_template/setup.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 with open('asv_test_repo/build_time_env.py', 'w') as f:
-    f.write("env = {{}}\n".format(repr(dict(os.environ))))
+    f.write(f'env = {os.environ}')
```
Comment (datapythonista) [inline test/test_repo_template/setup.py:6]:
```suggestion
    f.write('env = {}')
```

**Comment by datapythonista** [comment]
Can you split the changes here in two separate PRs please? Can't understand what's the problem, and having unrelated changes mixed doesn't help.

File: test/test_repo_template/setup.py (hunk 3 above)
Comment (dorothykiz1) [inline test/test_repo_template/setup.py:6]: looks like this we need to be specific on the key we want to return from this particular output that has environment varibles, hence `E                   KeyError: 'os' `

Comment (datapythonista) [inline test/test_repo_template/setup.py:6]:
```suggestion
    f.write('env = {}')
```

File: test/test_repo_template/setup.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 with open('asv_test_repo/build_time_env.py', 'w') as f:
-    f.write("env = {{}}\n".format(repr(dict(os.environ))))
+    f.write('env = {}')
```
Comment (datapythonista) [inline test/test_repo_template/setup.py:6]:
```suggestion
    f.write("env = {{}}\n".format(repr(dict(os.environ))))
```

**Comment by datapythonista** [comment]
Not sure what's going on with this. You can leave the file in the exclude list for now, or add a `noqa` with the error to ignore, and a link to an issue explaining the problem.

**Comment by datapythonista** [comment]
The file is already excluded, this doesn't make much sense if you don't remove `setup.py`, which is more generic and is also excluding other files with the same name.

And as said, if we're to exclude this file, add the issue with the problem to the exclude list.

**Comment by dorothykiz1** [comment]
Thanks i will do it

**Comment by datapythonista** [comment]
Superseded by #1160

**Pull request closed without merging**